### PR TITLE
test(topsql): init

### PR DESCRIPTION
**Pull request by shhdgit**
Add top sql tests

### CODE REVIEW

File: ui/cypress/integration/topsql/topsql.spec.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    })
+  })
+
+  describe('SQL statement details', () => {
+    it('click one table row, show the list detail table and information contents', () => {
+      setCustomTimeRange('2022-01-12 00:00:00{enter}2022-01-12 05:00:00{enter}')
+
+      cy.get('[data-e2e="topsql-list-table"] .ms-List-cell').eq(0).click()
+      cy.get('[data-e2e="topsql-listdetail-table"]').should('exist')
+      cy.get('.e2e-topsql-listdetail-content-sql_text').should('exist')
```
Comment (breeswish) [inline ui/cypress/integration/topsql/topsql.spec.js:203]: I see that some are using classes and some are using attributes. Are there ways to make them unified?

Comment (shhdgit) [reply ui/cypress/integration/topsql/topsql.spec.js:203]: Let me try.

**Comment by breeswish** [comment]
@YiniXu9506 PTAL~

File: ui/lib/apps/TopSQL/pages/List/ListDetail/ListDetailContent.tsx [hunk 2]
```diff
@@ -63,6 +64,7 @@ export function ListDetailContent({
           </Expand>
         </Descriptions.Item>
         <Descriptions.Item
+          className="sql_digest"
```
Comment (breeswish) [inline ui/lib/apps/TopSQL/pages/List/ListDetail/ListDetailContent.tsx:67]: It may be not a good idea to use className for e2e tests.

**Comment by shhdgit** [comment]
> @shhdgit Would you like to drive the merging of this PR when you are free? Thanks!

Yeah, sure!

File: scripts/start_tiup.sh [hunk 3]
```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 set -ex
 tidb_version=$1
-mode=$2
+without_ngm=${2:-false}
+mode=${3:-"start"}
```
Comment (YiniXu9506) [inline scripts/start_tiup.sh:5]: The order of args will cause failing to restart TiUP.

Comment (shhdgit) [reply scripts/start_tiup.sh:5]: Nice catch! Thanks!

**Review by breezewish — APPROVED** [review]
A question: could you write instructions about how to update the snapshot PNGs here?

**Comment by shhdgit** [comment]
> A question: could you write instructions about how to update the snapshot PNGs here?

Done.

**Pull request merged**